node/src/archive/moonlight.rs

```diff
@@ -36,7 +85,7 @@ impl Archive {
     ///
     /// * `path` - The path to the archive folder.
     /// * `archive_opts` - The options for the archive.
-    pub(super) async fn create_or_open_moonlight_db<
+    pub(super) fn create_or_open_moonlight_db<
```

Comment:
Does this need to be `async`? Seems to me there's no `await` called in the function.